local_planner/include/local_planner/local_planner_node.h

```diff
@@ -214,17 +203,18 @@ class LocalPlannerNode {
   void fillUnusedTrajectoryPoint(mavros_msgs::PositionTarget& point);
 
   /**
-  * @brief     publishes position and velocity setpoints both to the FCU and to
-  *Rviz for visualization
+  * @brief     calculateds position and velocity setpoints and sends to the FCU
```

Comment:
- `calculates`
- remove `and to RViiz for visualization`
- Fix awkward line break here